src/settings/Settings.tsx

```diff
@@ -804,6 +804,11 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         default: true,
         controller: new IncompatibleController("feature_breadcrumbs_v2", true),
     },
+    "FTUE.userOnboardingButton": {
```

Comment:
err, is this supposed to be a setting? Usually we avoid settings for this sort of stuff.

Reply:
Considering the user should be able to get the button back, if they wish to, and dismissing this shouldn’t be something they have to do on every single device again, I’m not entirely sure what alternative you’re suggesting.

Reply:
it's just not usually something we make a setting. We'd just always have this enabled based on the language.

Critically, this is not for the welcome page (an existing concept) - it's for a new *intro* page.

Reply:
> it's just not usually something we make a setting. We'd just always have this enabled based on the language.

Then please discuss that with design and product, because they’ve specified it to be removable as a setting. They also used the term "Welcome Home Screen", though I personally prefer "user onboarding".

See the following epic for decisions on this: https://github.com/vector-im/element-web/issues/22463